packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -205,7 +198,9 @@ const FooterTagList = ({
     setDisplayShowAllButton(false);
   }, [setShowAll, setDisplayShowAllButton]);
 
-  const tagIds = (results||[]).map((tag) => tag._id)
+  const tagIds = (results ? results.map((tagRel) => tagRel.tag?._id) : post.tags.map((tag) => tag._id)).filter(
+    Boolean
+  ) as string[];
```

Comment:
Unrelated bug: these where tagRel ids rather than tag ids, so this didn't remove the already selected tags from the core tags list as intended